tools/src/main/java/org/apache/kafka/tools/ConsumerPerformance.java

```diff
@@ -335,8 +349,18 @@ public ConsumerPerfOptions(String[] args) {
             }
             if (options != null) {
```

Comment:
The KIP also adds `--command-property` to both `kafka-consumer-perf-test.sh` and `kafka-share-consumer-perf-test.sh`.

Reply:
No, I don't think so.

I thought about that when I wrote the KIP and I decided not to replicate across the other performance tools. In all of the command-line tools except `kafka-producer-perf-test.sh`, the `--bootstrap-server` (or `--bootstrap-conteroller`) flag is mandatory and people are used to that. This KIP improves the consistency of the producer perf test, but it also works with the previous way of working.

If we are to start making `--bootstrap-server` optional more broadly so people can use the config file to specify it instead, I would say that should be allowed in all of the tools, and we'd need to work out what to do about `--bootstrap-controller`. Personally, I don't think that's worth the effort, and it would definitely be another KIP.